src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Parser;
+import duke.StorageList;
+import duke.Ui;
+
+/**
+ * Add a new Task to the StorageList.
+ */
+public class AddCommand extends Command {
+    /**
+     * Executes the add command.
+     *
+     * @param ui the user interface
+     * @param storageList the storage list
+     * @throws DukeException if the command is invalid
+     */
```

Comment:
I feel that it would be tidier and more efficient to put Javadoc comment for just the overridden method in the abstract command class as it can potentially save a lot of your time. (javadoc comments still applies for overriding method)

Reply:
Thanks, will do so for future overriden methods!